karma.conf.js

```diff
@@ -12,7 +12,7 @@ module.exports = function(config) {
     files: [testGlob, srcGlob],
     exclude: ['decidim-*/app/frontend/entry.js'],
     preprocessors: {
-      [testGlob]: ['webpack'],
+      [testGlob]: ['webpack', 'sourcemap'],
```

Comment:
That's just for karma don't worry. It's true I'm creating the source map in the production build but I'm not including it in sprockets so it's no used right now.